extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -132,3 +140,7 @@ export function getFetcherClassForTesting() {
 export function getPageConfigClassForTesting() {
   return PageConfig;
 }
+
+export function getServiceAdapterClassForTesting() {
```

Comment:
This shouldn't be needed. Instead, please use the relative expanded type. I know it's confusing, but that `getPageConfigClassForTesting` is only needed at this time for third_party libraries.